### ISSUE TITLE
v1.1.1 failed call to cuInit: UNKNOWN ERROR (303)

### ISSUE THREAD

**Issue by Psy-Fer** [nanoporetech/medaka#214]
**Describe the bug**
Getting this error...but it's not killing the job? not really sure if this is a bad thing or no?

**Logging**
[23:28:14 - Predict] Setting tensorflow threads to 32.
2020-10-08 23:28:14.726991: E tensorflow/stream_executor/cuda/cuda_driver.cc:313] failed call to cuInit: UNKNOWN ERROR (303)
[23:28:14 - Predict] Processing 1 long region(s) with batching.

**Environment:
 - Installation method [conda]
 - OS: [e.g. CentoOS 7]
 - No GPU

**Additional context**
Running on coronavirus samples.

Just double checking output.

Cheers,
James

**Comment by cjw85** [comment]
Eeek. I've not seen this one. Do you have CUDA installed despite not using a GPU?

**Comment by Psy-Fer** [comment]
Nope

Was running on one of our CPU only nodes. Only our GPGPU nodes have any CUDA.

I checked:

`lspci | grep -i nvidia `
`cat /proc/driver/nvidia/version`
`nvcc -V`


no results for anything.

**Comment by cjw85** [comment]
Seems like its this issue, and harmless. One might expect to fail to call `cuInit()` if CUDA isn't present.

https://github.com/tensorflow/tensorflow/issues/37689

**Comment by cjw85** [comment]
Just some background before I close this.

The python packages for tensorflow used to come in two flavours: `tensorflow` and `tensorflow-gpu`. Using a GPU was sort of an opt in thing, the latter package would not function if you didn't have a GPU and CUDA installed. We decided to make medaka require the former of these as we anticated most users wouldn't have a GPU.

The situation presently is that there are still two tensorflow packages: `tensorflow` and `tensorflow-cpu`. The former now allows use of GPUs but crucially doesn't break things for non-GPU users. Its therefore the preferred option as a generic requirement. The second package is in essence equivalent to the old `tensorflow` package and only enables use of CPU; the benefit of using it would be to avoid spurious errors like the one in this issue.
